recipes/libavrocpp/config.yml

```diff
@@ -0,0 +1,3 @@
+versions:
+  1.10.1:
+    folder: 1.10.1
```

Comment:
I can not see this project change so drastically that a new recipe is required for each patch release 

Please more the recipe to an `all` folder